packages/client/clients/internal/talk-client.ts

```diff
@@ -310,4 +310,206 @@ export class TalkClient extends ChannelClient {
 			this.TalkService_API_PATH,
 		);
 	}
+
```

Comment:
```suggestion
/**
 * @description Kick out members of the chat.
 */
```

Please add a brief description in jsdoc like this.
Also run `deno run format` to keep the code in the same format.